test/unit/ode/test_ode.py

```diff
@@ -100,6 +100,16 @@ def test_differentiate2c():
         "g",
     )
 
+    assert _equivalent(
+        differentiate2c(
+            "-f(x)",
+            "x",
+            {},
+        ),
+        "1000.0*f(x - 0.00050000000000000001) - 1000.0*f(x + 0.00050000000000000001)",
```

Comment:
Is there a more robust way of comparing the equality here? The SymPy codgen can change at some version and the order of the operands could change, making this test rather brittle.

Reply:
Yes, you make it generate a function that you can run.

Reply:
Feeding the output back to SymPy seems to work.